From 3917b152e97693c68e68ec4747f569843f5706a2 Mon Sep 17 00:00:00 2001
From: Philippe Renon <philippe_renon@yahoo.fr>
Date: Fri, 16 Mar 2018 17:04:37 +0100
Subject: [PATCH] msys2: fix jsoncpp related link failure

found jsoncpp library was not added to SERVER_EXTRA_LIBS
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d79ee4c6..e0d69245 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -547,7 +547,7 @@ option_requires(VRPN_USE_JSONNET
 
 if(VRPN_USE_JSONNET)
 	include_directories(BEFORE ${JSONCPP_INCLUDE_DIRS})
-	list(APPEND SERVER_EXTRA_LIBS ${JSONCPP_LIBRARIES})
+	list(APPEND SERVER_EXTRA_LIBS ${JSONCPP_LIBRARY})
 else()
 	set(JSONCPP_SOURCES)
 endif()
-- 
2.16.2

